From 6cb8b2503e8102d963e736cab497e6f7c5f1e991 Mon Sep 17 00:00:00 2001
From: Felipe Leme <felipeal@google.com>
Date: Mon, 19 Mar 2018 11:15:00 -0700
Subject: [PATCH 161/306] Fixed AutofillManager.isAutofillUiShowing()

It was using & instead of &&

Test: not really
Bug: 75652829
Bug: 74527545

Change-Id: Ib8ca79ba6ec35d2fc97f042db2152b03d334ce30
(cherry picked from commit cb2e83da36710b8f6a043ccc9ac30249003d7950)
---
 core/java/android/view/autofill/AutofillManager.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/view/autofill/AutofillManager.java b/core/java/android/view/autofill/AutofillManager.java
index c1092978cea..b55734da9bd 100644
--- a/core/java/android/view/autofill/AutofillManager.java
+++ b/core/java/android/view/autofill/AutofillManager.java
@@ -1413,7 +1413,7 @@ public final class AutofillManager {
      */
     public boolean isAutofillUiShowing() {
         final AutofillClient client = mContext.getAutofillClient();
-        return client != null & client.autofillClientIsFillUiShowing();
+        return client != null && client.autofillClientIsFillUiShowing();
     }
 
     /** @hide */
-- 
2.17.1

